src/web/client/common/worker/webworker.js

```diff
@@ -82,6 +82,11 @@ let intital = false;
 
 async function loadContainer(config, swpId, entityInfo) {
     try {
+        self.postMessage({
```

Comment:
Are they using webextensiontelemetry ? If yes then we should be good. If not then what is the data residency part for this scenarios

Reply:
yes they uses webExtensionTelemetry [link](https://github.com/microsoft/powerplatform-vscode/pull/824/files#:~:text=if%20(data.type%20%3D%3D%3D%20Constants.workerEventMessages.SEND_INFO_TELEMETRY)%20%7B)